src/components/collabs/pick.svelte

```diff
@@ -27,11 +27,25 @@
 			return;
 		}
 
-		const response = _window.confirm($t('collabs.delete_pick_confirm'));
+		let confirmed = false;
+		let reason: string | null = '';
 
-		if (response) {
-			await axios.delete('/api/collabs/' + collab.id + '/picks/' + pick.id);
+		if ($discord?.admin && pick.userId !== $discord?.id) {
+			reason = _window.prompt('Whats the reason for the deletion?', 'Duplicate Pick');
+			if (reason) {
+				confirmed = true;
+			}
+		} else if (pick.userId === $discord?.id) {
```

Comment:
Not the change I meant regarding the if statement. Else ifs tend to make program flows a little harder to read. Add a new if in the admin check.

Reply:
Removed the else if and put it into an if by itself.